framework/tlvf/test/main.cpp

```diff
@@ -152,6 +153,49 @@ int main(int argc, char *argv[])
     LOG(DEBUG) << "Done (WSC M2)";
     MAPF_INFO("TLV LENGTH WSC M2: " << thirdTlv->length());
 
+    auto fourthTlv     = msg.addClass<tlvTestVarList>();
+    fourthTlv->var0()  = 0xa0;
+    allocation_succeed = fourthTlv->alloc_simple_list(2);
+    mapf_assert(allocation_succeed);
```

Comment:
Perhaps it would be better to align all tests to the way you wrote the encryption_test, with the `check` function.
Using assert for some failures and incrementing errors on others is inconsistent.

Reply:
Agreed, but it's a copy from earlier.